docs/design_principles.md

```diff
@@ -0,0 +1,76 @@
+<style>
+    .spacing {
+        margin-top: 36px;
+    }
+</style>
+
+
+<div class="spacing">
+	<p>Principles are the foundation of the system. They form the basis of a good product and help the team with decision making. They are here to guide you and your team when working with the myriad parts of the system and help you do better and make more informed decisions.</p>
+</div>
+
+
+<div class="spacing">
+    <h3>###1. We start with user needs.</h3>
```

Comment:
Remove div class spacing, `<h3> tag and <hr>`. Use only Markdown syntax. In this case: `## We start with user needs` it will create ah h2 heading.

Reply:
The same for the rest. We only need the markdown syntax not html.

Reply:
Take a look on this [markdown from the old lux](https://raw.githubusercontent.com/pulibrary/lux/refs/heads/main/docs/principles.md)

In your PR though you need to have an h1 on the top. Which means adding `# Design Principles` as the title of the page. The rest would be as I previously commented `h2` so that there is a sequence between the headers.